lib/zypper.rb

```diff
@@ -91,8 +91,6 @@ def version
     [found[1].to_i, found[2].to_i, found[3].to_i] if found
   end
 
-  private
```

Comment:
What makes this obsolete? I think it still makes sense to restrict access to the actual `call_zypper` method.